vcluster/configure/vcluster-yaml/sleep-mode.mdx

```diff
@@ -0,0 +1,196 @@
+
+```yaml title="sleep mode configuration"
+sleepMode:
+  enabled: true
+  autoSleep:
+    afterInactivity: 1h
+```
+
+## How it works
+
+### Sleep mode operations
+
+Sleep mode performs two main operations:
+
+**Sleeping**
+Deletes bare pods and scales down the following resources:
+- Deployments
+- ReplicaSets
+- ReplicationControllers
+- DaemonSets
+
+**Waking**
+
+- Scales resources back to their original state
+- Cannot restore previously deleted bare pods
+
+### Resource exemption
+
+Resources can be exempted from sleep mode using either:
+- Adding the annotation `sleepmode.loft.sh/exclude: true`
+- Configuring `sleepMode` with specific labels
+- Adding configured labels to workloads that should keep running
+
+## Detecting activity
+
+To wake a sleeping cluster or to update the last active time, sleep mode captures the following:
+
+- Access of cluster resources through API calls (e.g., `kubectl get <resource>`)
+- Attempts to contact Ingress endpoints (NGINX only)
+
+:::note
+Ingress activity detection is only available for [NGINX](https://github.com/kubernetes/ingress-nginx/tree/main)
+ingress controllers, making use of the [mirror-target](https://github.com/kubernetes/ingress-nginx/blob/main/docs/user-guide/nginx-configuration/annotations.md#mirror)
+annotation.  This has the effect of overwriting any previously set mirror-target annotation.
+:::
+
+### Ignoring other types of activity
+
+| Annotation| Behavior | Format |
+|-----------|----------|--------|
+| "sleepmode.loft.sh/ignore-all" | ignore all activity | "true" OR "false" |
+| "sleepmode.loft.sh/ignore-ingresses" | ignore requests to associated ingresses  | "true" OR "false"  |
+| "sleepmode.loft.sh/ignore-groups" | ignore requests with a specific apigroup | "apps,group2,group3" |
+| "sleepmode.loft.sh/ignore-resources" | ignore requests to specific resource types  | "pods,resource2,resource3"  |
+| "sleepmode.loft.sh/ignore-verbs" | ignore requests with specific verbs | "create,verb1,verb2"  |
+| "sleepmode.loft.sh/ignore-resource-verbs" | ignore requests with specific verbs to specific resources |  "myresource.mygroup=create update delete, myresource2.mygroup=create update" |
+| "sleepmode.loft.sh/ignore-resource-names" | ignore requests to specific resources with specific names | "myresource.mygroup=name1 name2,myotherresource.mygroup=name1"  |
+| "sleepmode.loft.sh/ignore-user-agents" | ignore specific useragents with trailling wildcard support | "kubectl*,argo,useragent3"  |
+
+:::note
+These annotations are to be added to the vCluster workload, e.g. the `StatefulSet` or `Deployment` running the virtual cluster, and any request can be ignored dynamically by adding the header `X-Sleep-Mode-Ignore`.
+:::
+
+## Differences and compatibility with the platform
+
+The documentation on this page describes sleep mode for a vCluster without an agent, where workloads are scaled down, but the control plane remains active to monitor for activity that should wake it up.
```

Comment:
I would pivot this to instead describe the two ways it could run, something like: 
```suggestion
The documentation on this page describes two primary methods for sleep mode. The first is without an agent, where workloads are scaled down, but the control plane remains active to monitor for activity that should wake it up. And second... (sentence below) 
```
Because this page is really about both ways